draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -707,20 +706,20 @@ using the MP_FAST_CLOSE suboption.
 ~~~~
 {: #ref-MP_FAST_CLOSE title='Format of the MP_FAST_CLOSE Suboption'}
 
-The MP_FAST_CLOSE suboption MUST be sent from an initiating host on all subflows 
-using a DCCP-Reset packet with Reset Code THIS-RESET-CODE. 
-To protect from unauthorized shutdown of a multipath DCCP connection, 
+When host A wants to abruptly close MP-DCCP connection with host B, it will send out the MP_FAST_CLOSE. The MP_FAST_CLOSE suboption MUST be sent from host A on all subflows 
```

Comment:
```suggestion
When host A wants to abruptly close an MP-DCCP connection with host B, it will send out the MP_FAST_CLOSE. The MP_FAST_CLOSE suboption MUST be sent from host A on all subflows 
```